ml-agents/mlagents/trainers/ppo/trainer.py

```diff
@@ -461,11 +473,13 @@ def update_policy(self):
             for l in range(
                 len(self.training_buffer.update_buffer["actions"]) // n_sequences
             ):
-                start = l * n_sequences
-                end = (l + 1) * n_sequences
-                run_out = self.policy.update(
-                    buffer.make_mini_batch(start, end), n_sequences
-                )
+                mini_batches = []
+                n_sequences_per_device = n_sequences // len(self.devices)
```

Comment:
Is there a way we could keep the `Trainer` class from knowing about devices?  Seems like we're still passing the same number of sequences total, so we could probably do this split at the `Policy` level.